src/_locales/ur/messages.json

```
@@ -0,0 +1,429 @@
+      "description": "This is an option shown in the add-on settings."
+    },
+  
+    "optionDebugMode": {
+      "message": "ڈیبگ موڈ اون کرو۔",
+      "description": "This is an option shown in the add-on settings."
+    },
+    "optionDebugModeDescr": {
+      "message": "یہ سرف کونسول مے سے کچ معلومات نکالے اور ہمیں دکھانے کے لیے ہے۔",
+      "description": "The description shown for the debug mode (optionDebugMode)."
+    },
+  
+    "translatorCredit": {
+      "message": "اس پروجکٹ کو $TRANSLATORS$ نے اردو مے ترجما کیا ہے۔.",
+      "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+      "placeholders": {
+        "translators": {
+          "content": "$1",
+          "example": "<a href=\"https://github.com/Toprun123/\">@Toprun123</a>"
+        }
+      }
+    },
+    "translatorLink": {
+      "message": "https://github.com/Toprun123",
+      "description": "The link to the translator's GitHub profile."
+    },
+    "translatorUsername": {
+      "message": "Toprun123",
+      "description": "The username that the translator wants to be referred to."
+    },
+  
+    "contributorsThanks": {
+      "message": "اور شکریا ان سب کو: $CONTRIBUTORS$.",
+      "description": "Text thanking all contributors and linking to the contributors file.",
+      "placeholders": {
+        "contributors": {
+          "content": "$1",
+          "example": "<a href=\"https://github.com/rugk/…/CONTRIBUTORS.md\">all other contributors</a>"
+        }
+      }
+    },
+    "contributorsThanksLink": {
+      "message": "https://github.com/rugk/offline-qr-code/blob/master/CONTRIBUTORS.md",
+      "description": "The link to the CONTRIBUTORS file."
+    },
+    "contributorsThanksLinkText": {
+      "message": "اور باقی سب جنہنے مدد کی۔",
+      "description": "The link text linking to the contributors file. See contributorsThanks."
+    },
+  
+    // ARIA labels/descriptions
+    "dismissIconDescription": {
+      "message": "بند کریں اسے۔",
+      "description": "the aria label for the close button of the message box"
+    },
+    "ariaMessageLoading": {
+      "message": "لوڈ ہو رہا ہے۔",
+      "description": "the aria label to label the message box as a loading message box"
+    },
+    "ariaMessageInfo": {
+      "message": "مالومات",
+      "description": "the aria label to label the message box as an info message box"
+    },
+    "ariaMessageSuccess": {
+      "message": "کامیاب",
+      "description": "the aria label to label the message box as an success message box"
+    },
+    "ariaMessageError": {
+      "message": "مسلہ",
+      "description": "the aria label to label the message box as an error message box"
+    },
+    "ariaMessageWarning": {
+      "message": "انتباہ",
+      "description": "the aria label to label the message box as an warning message box"
+    }
+  }
+  
```

```suggestion
}
```

FYI as far as I see this is the correct indentation?

---

I've indented the whole thing half half based on 2 and 4 spaces by mistake.. I've changed it now to follow a 2 space indentation throughout.